spec/spec_helper.rb

```diff
@@ -2,7 +2,6 @@
 
 require 'rspec_api_docs'
 require 'rspec_api_docs/after'
-require 'pry'
```

Comment:
I think I'd like to keep this in the spec helper so it's available when testing.